e2e/pages/events/manageDocumentsEvent.page.js

```diff
@@ -114,4 +122,9 @@ module.exports = {
     await this.uploadSupportingEvidenceDocument(supportingEvidenceDocument, selectEvidenceType);
     this.selectConfidential(index);
   },
+
+  async selectTranslationRequirement(index, requirement) {
```

Comment:
doesn't look like this needs to be async

Reply:
👍